x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/header_actions/search_engines_playground.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiButton } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { ENTERPRISE_SEARCH_CONTENT_PLUGIN } from '../../../../../../../common/constants';
+import { KibanaLogic } from '../../../../../shared/kibana';
+
+import { PLAYGROUND_PATH } from '../../../../routes';
+
+export interface SearchEnginesPlaygroundProps {
+  indexName: string;
+  ingestionMethod: string;
+}
+
+export const SearchEnginesPlayground: React.FC<SearchEnginesPlaygroundProps> = ({
```

Comment:
This should probably be called SearchPlaygroundPopover, not SearchEnginesPlayground--it's not a playground for search engines.

Reply:
Agreed - we want to try keep away from the "Engine" name. SearchPlaygroundPopover is good.